lib/src/lifecycle_module.dart

```diff
@@ -1161,18 +1163,18 @@ class ModuleUnloadCanceledException implements Exception {
 /// A set of messages returned from the hierarchical application of shouldUnload
 class ShouldUnloadResult {
   bool shouldUnload;
-  List<String> messages;
+  List<String>? messages;
```

Comment:
```suggestion
  List<String> messages = [];
```
Making this non-nullable will allow removing a lot of `.messages!` in this PR